tgmc.dme

```diff
@@ -12,6 +12,7 @@
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
 #include "_maps\map_files\generic\Admin_Level.dmm"
+#include "_maps\map_files\Theseus\TGS_Theseus.dmm"

```

Comment:
This shouldn't be included.

Reply:
Aren't these the map changes? How am i not to include them

Reply:
They are dynamically loaded by the SSmapping subsystem